vignettes/articles/glitter_for_Wikidata.Rmd

```diff
@@ -18,178 +18,225 @@ knitr::opts_chunk$set(
+To find the identifiers of items and properties of interest for a particular case study, you can:
 
-# Example 1: Lyon Metro
+- browse [Wikidata](https://www.wikidata.org/wiki/Wikidata:Main_Page)
+- use package `WikidataR` (functions `find_item()`,`find_property()`). Here, we will explore that second option
 
-Imagine you are interested in the Wikidata available regarding the **Lyon Metro network**.
-
-Let's try and see if there are Wikidata about it:
+Let's try and find the Wikidata identifier for the Lyon metro network:
 
 ```{r}
-find_item("Metro Lyon")
+WikidataR::find_item("Metro Lyon")
 ```
 
-So you'd be interested, for instance, in all the subway stations that are part of this network.
+So you'd be interested, for instance, in all the subway stations that are part of this network. Let's try and find the property identifier that corresponds to this notion:
 
 ```{r}
-find_property("part of")
+WikidataR::find_property("part of")
 ```
 
-So you're looking for all the stations that are part of ("wd:P361") the Lyon metro network ("wd:Q1552").
+So you're looking for all the stations that are part of ("wdt:P16") the Lyon metro network ("wd:Q1552").
 
-You could access this information through:
+# Use glitter functions to start exploring data
 
-```{r}
+## Simple single-line query
+
+The `glitter` functions might now be used to start exploring data.
+
+We're looking for items (the "unknown" in our query below, hence the use of a "?") which are part of the Lyon metro network:
+
+```{r single_line}
 stations=get_triple("?items wdt:P16 wd:Q1552")
-stations %>% head()
+head(stations)
 ```
 
-Notice that we do not have values yet for the stations (that's what we're looking for) hence the use of "?" at the beginning of the subject string.
-
 To also get the labels for stations, we can use the argument `label`:
 
-```{r}
-parts_metro_Lyon=get_triple("?items wdt:P16 wd:Q1552", label="?items")
-parts_metro_Lyon %>% head()
+```{r single_line_with_label}
+parts_metro_Lyon=get_triple("?items wdt:P16 wd:Q1552",
+                            label="items")
+head(parts_metro_Lyon)
 ```
 
-For now, we get 50 items, not only stations but also other types of items such as metro lines. Let's have a look at the item "Place Guichard - Bourse du Travail" (Q59855) which we know correspond to a station.
-
-Property "wdt:P31" should enable us to collect only stations ("wd:Q928830") instead of all parts of the Lyon Metro network.
+## Simple multi-line query
 
-We can enrich and refine our query incrementally with `spq_filter()` before sending it (whereas `get_triple()` was a shortcut to build and send simple requests) :
+The `get_triple()` function we used above is a good option when you want to send **a very simple query**, yet more often than not one will wish to **gradually complexify** their initial query, so that it's actually preferable to use the **multi-line equivalent** to get_triple, which allows for such increments:
```

Comment:
should we show get_triple at all?

Reply:
indeed. I never use qplot because by now I never remember exactly how it works. Maybe we should not show it. (Very normande answer)

Reply:
I think it'd be better not to show it indeed :innocent: 

> qplot() is now deprecated in order to encourage the users to learn [ggplot()](https://ggplot2.tidyverse.org/reference/ggplot.html) as it makes it easier to create complex graphics.

https://ggplot2.tidyverse.org/reference/qplot.html